doc/Makefile.in

```diff
@@ -18,27 +16,15 @@ VPATH = @srcdir@
 
 version = $(shell cat $(top_srcdir)/VERSION)
```

Comment:
Hmm good point. I guess I could re-introduce it for when the docs are generated. Right now it is in "manual mode" in the Doxyfile. Will fix!